model-optimizer/extensions/ops/sparse_reshape.py

```diff
@@ -3,13 +3,15 @@
 
 import numpy as np
 
+from mo.front.common.partial_infer.utils import dynamic_dimension, is_fully_defined, shape_array, strict_compare_tensors, dynamic_dimension_value
 from mo.graph.graph import Node, Graph
 from mo.ops.op import Op
 
 
 class SparseReshape(Op):
     """
-    SparseReshape operation reshapes a sparse tensor. It recomputes indices for a new dense shape.
+    SparseReshape operation reshapes a sparse tensor in COO format (https://en.wikipedia.org/wiki/Sparse_matrix#Coordinate_list_(COO))
```

Comment:
please do not add a link to wiki. It locks not good. Just say that `... a sparse tensor in Coordinate list format`.
If someone does not know it, he can easily find it.

Reply:
removed link to Wiki